securesystemslib/interface.py

```diff
@@ -938,7 +931,12 @@ def import_ecdsa_privatekey_from_file(filepath, password=None,
       password)
 
   # Raise an exception if an unexpected key type is imported.
-  if key_object['keytype'] != 'ecdsa-sha2-nistp256':
+  # NOTE: we support keytype's of ecdsa-sha2-nistp256 and ecdsa-sha2-nistp384
+  # in order to support key files generated with older versions of
+  # securesystemslib. At some point this backwards compatibility should be
+  # removed.
+  if key_object['keytype'] not in['ecdsa', 'ecdsa-sha2-nistp256',
```

Comment:
Should this be in in-toto-golang, too?

CC: @lukpueh

Reply:
I don't think that's necessary. Just wanted to loop you in.

Reply:
I don't think that's necessary. Just wanted to loop you in.

Reply:
Good! :)